README.md

```diff
@@ -198,7 +198,7 @@ The default configuration can be found in `config/default.json`.
   "tracker": { // Tracking mechanism to create GitHub issues when document content is inaccessible
     "githubIssues": {
       "repository": "GitHub repository where to create isssues",
-      "label": "GitHub issues label to attach to a newly created issue. This specific label has to exist in the repository",
+      "label": "GitHub issues label to attach to a newly created issue. This specific label will be created automatically in the target repository",
```

Comment:
Can you use this PR to change the configuration like this:
```
"tracker": {
    "githubIssues": {
      "repository": "OpenTermsArchive/sandbox",
      "label": {
        "name": "bot-report",
        "color": "FEF2C0",
        "description": "Automatically created by Open Terms Archive when fetching error is encountered"
      }
    }
  },
  ```

Reply:
Done @Ndpnt I also renamed the variables to be more coherent

Reply:
You forgot to update the README file.

Reply:
That was quite a huge scope creep. I agree with the change, and I don't think it will have major consequences from what we know of current reusers, but let's please keep in mind that any change to the config can break down instances that don't update their config accordingly and be careful about it 🙂